src/examples/unique-taxpayer-reference/corporation-tax/index.njk

```diff
@@ -7,7 +7,10 @@ layout: layout-example.njk
 
 <h1 class="govuk-heading-l">What is your Corporation Tax Unique Taxpayer Reference (UTR)?</h1>
 
-<p class="govuk-body">You can <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.gov.uk/find-lost-utr-number">find a lost UTR (opens in new tab)</a>.</p>
+<p class="govuk-body"><a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.gov.uk/find-lost-utr-number">Find a lost UTR (opens in new tab)</a>.</p>
+
+<!-- The below line of code is a subsititute for the empty hint: html: text below which is where this text should be but unsure how to add a link there -->
```

Comment:
I think you should just be able to add the HTML in the `hint: {html: ''}` object. I noticed the self assessment example file is using the text String option but you should just be able to swap it out for the html String option instead.

If you go to https://design-system.service.gov.uk/components/text-input/, click the Nunjucks tab on the first example and then click 'Nunjucks macro options' you'll see all the different options you can pass in. If you scroll down to 'Options for hint' you'll see some info on how text/html work.

One thing to watch out for is that if you're using single quotes to open and close the html String option i.e. `html: ''` make sure the HTML that you pass in isn't using single quotes for attributes otherwise Nunjucks will think you're trying to close the String.